extensions/wikia/ArticleVideo/styles/jwplayer.scss

```diff
@@ -0,0 +1,105 @@
+@import 'video-shared-variables';
+
+.featured-video {
+	padding-top: 56.25%; // we want to have 16:9 ratio div size
+	position: relative;
+	width: 100%;
+
+	&__player-container {
+		bottom: 0;
+		left: 0;
+		position: absolute;
+		right: 0;
+		top: 0;
+		transform: translate3d(0, 0, 0); // hack for iOS for changing position to fixed while scrolling
+		z-index: 2;
+	}
+
+	&.is-collapsed-ready &__player-container {
+		left: auto;
+		position: fixed;
+		top: auto;
+	}
+
+	&.is-collapsed &__player-container {
```

Comment:
Maybe we could open &.is-collapsed once and then nest all the subselectors inside it :)